applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1616,60 +1610,60 @@ void SmallStrainUPwDiffOrderElement::InitializeElementVariables(ElementVariables
     KRATOS_CATCH("")
 }
 
-void SmallStrainUPwDiffOrderElement::InitializeNodalVariables( ElementVariables& rVariables )
+void SmallStrainUPwDiffOrderElement::InitializeNodalVariables(ElementVariables& rVariables)
 {
     KRATOS_TRY
 
     const GeometryType& rGeom = GetGeometry();
-    const SizeType Dim = rGeom.WorkingSpaceDimension();
-    const SizeType NumUNodes = rGeom.PointsNumber();
-    const SizeType NumPNodes = mpPressureGeometry->PointsNumber();
+    const SizeType Dim        = rGeom.WorkingSpaceDimension();
+    const SizeType NumUNodes  = rGeom.PointsNumber();
+    const SizeType NumPNodes  = mpPressureGeometry->PointsNumber();
 
-    Vector BodyAccelerationAux    = ZeroVector(3);
-    rVariables.BodyAcceleration.resize(NumUNodes * Dim,false);
-    rVariables.DisplacementVector.resize(NumUNodes * Dim,false);
-    rVariables.VelocityVector.resize(NumUNodes * Dim,false);
+    Vector BodyAccelerationAux = ZeroVector(3);
+    rVariables.BodyAcceleration.resize(NumUNodes * Dim, false);
+    rVariables.DisplacementVector.resize(NumUNodes * Dim, false);
+    rVariables.VelocityVector.resize(NumUNodes * Dim, false);
 
-    for (SizeType i=0; i<NumUNodes; ++i) {
-        SizeType Local_i = i * Dim;
+    for (SizeType i = 0; i < NumUNodes; ++i) {
+        SizeType Local_i    = i * Dim;
         BodyAccelerationAux = rGeom[i].FastGetSolutionStepValue(VOLUME_ACCELERATION);
 
         rVariables.BodyAcceleration[Local_i]   = BodyAccelerationAux[0];
         rVariables.DisplacementVector[Local_i] = rGeom[i].FastGetSolutionStepValue(DISPLACEMENT_X);
         rVariables.VelocityVector[Local_i]     = rGeom[i].FastGetSolutionStepValue(VELOCITY_X);
 
-        rVariables.BodyAcceleration[Local_i+1]   = BodyAccelerationAux[1];
-        rVariables.DisplacementVector[Local_i+1] = rGeom[i].FastGetSolutionStepValue(DISPLACEMENT_Y);
-        rVariables.VelocityVector[Local_i+1]     = rGeom[i].FastGetSolutionStepValue(VELOCITY_Y);
+        rVariables.BodyAcceleration[Local_i + 1] = BodyAccelerationAux[1];
+        rVariables.DisplacementVector[Local_i + 1] = rGeom[i].FastGetSolutionStepValue(DISPLACEMENT_Y);
+        rVariables.VelocityVector[Local_i + 1] = rGeom[i].FastGetSolutionStepValue(VELOCITY_Y);
 
-        if (Dim > 2 ) {
-            rVariables.BodyAcceleration[Local_i+2]   = BodyAccelerationAux[2];
-            rVariables.DisplacementVector[Local_i+2] = rGeom[i].FastGetSolutionStepValue(DISPLACEMENT_Z);
-            rVariables.VelocityVector[Local_i+2]     = rGeom[i].FastGetSolutionStepValue(VELOCITY_Z);
+        if (Dim > 2) {
+            rVariables.BodyAcceleration[Local_i + 2] = BodyAccelerationAux[2];
+            rVariables.DisplacementVector[Local_i + 2] = rGeom[i].FastGetSolutionStepValue(DISPLACEMENT_Z);
+            rVariables.VelocityVector[Local_i + 2] = rGeom[i].FastGetSolutionStepValue(VELOCITY_Z);
         }
```

Comment:
For some reason, the = signs do no longer align.

Reply:
Because of the extra spaces, aligning is no longer possible, since it would become more than 100 characters

Reply:
I think this is actually a bug in clang-format. For some reason, it does not take into account lines which overflow. For example, this
```
    shortname = 0;
    slightlyLongerName = 0;
    veryLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongName = 0;
```

Is nicely formatted to
```
    shortname                                                                                = 0;
    slightlyLongerName                                                                       = 0;
    veryLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongName = 0;
```

However, this:
```
    shortname = 0;
    slightlyLongerName = 0;
    veryLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongName = 20000000;
```

will be formatted into:
```
    shortname          = 0;
    slightlyLongerName = 0;
    veryLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongName = 20000000;
```

It makes sense that the column limit is taken into account, but here it seems its too strict and does not take into account that the rhs of an assignment could have a different length, such that this is done correctly